src/Services/src/DotnetCli/DotnetCliService.cs

```diff
@@ -44,6 +44,7 @@ public async Task<bool> PublishProjectForRemoteDebuggingAsync(string projectDir,
                     Directory.Delete(publishDirPath, true); // clean before recreating
                 }
                 var publishArgs = $"publish -f {targetFrameworkMoniker} -r {runtimeIdentifier} -c {configuration} -o {outputDirName} --self-contained";
+                StdOutCallback.Invoke($"Executing \"dotnet {publishArgs}\"...");
```

Comment:
add a space before ellipsis?
```
 {publishArgs}\" ..."
```

Reply:
Added in [4876248](https://github.com/vmware-tanzu/tanzu-toolkit-for-visual-studio/pull/345/commits/4876248a390289b844fa369ea76fe46fa05a1f23)